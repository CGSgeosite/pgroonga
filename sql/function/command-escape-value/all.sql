SELECT pgroonga.command_escape_value('a+B-c" \\あ\nいう');
