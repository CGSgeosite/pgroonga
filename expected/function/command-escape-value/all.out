SELECT pgroonga.command_escape_value('a+B-c" \\あ\nいう');
  command_escape_value   
-------------------------
 "a+B-c\" \\\\あ\\nいう"
(1 row)

